CHANGELOG.md

```diff
@@ -37,3 +43,6 @@
 
 - Change how stack [layouts are measured](https://github.com/square/Blueprint/pull/68) to resolve an issue where text would be truncated.
 - [Properly measure](https://github.com/square/Blueprint/pull/73) the child of `ScrollView` to allow for unconstrained measurement.
+
+[0.10.0]: https://github.com/square/Blueprint/compare/0.9.2...0.10.0
```

Comment:
This seems like it's in the wrong spot? (Probably from my updates from 0.9.2 where I added a couple missing releases... sorry!)

Reply:
Nope, this was intentional! Putting the URLs for these links at the bottom instead of inline keeps the text above a bit more readable, and it renders the same!

Reply:
👍

Reply:
(TIL!)